service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/documents/ManageDocumentsController.java

```diff
@@ -298,6 +300,10 @@ private DocumentUploaderType getUploaderType(Long id) {
             return DocumentUploaderType.SOLICITOR;
         }
 
+        if (caseRoles.stream().anyMatch(barristers()::contains)) {
+            return DocumentUploaderType.BARRISTER;
+        }
+
```

Comment:
Is it possible to move them before line 285? And the change of line 286 will not be needed.